### ISSUE TITLE
Linux, can't access out.log after 'npm install -g'

### ISSUE THREAD

**Issue by keforbes** [extr0py/oni#261]
This is specific to Linux but I don't think it's specific to my environment.  `npm install -g` requires superuser permission to install into `/usr/lib/` so I have to run `sudo npm install -g oni-vim`.  This is fine, it's expected of any `npm -g` command.

The problem is that after installing oni, it tries writing to a log file which still requires superuser permission.
```
Error: EACCES: permission denied, open '/usr/lib/node_modules/oni-vim/cli/out.log'
```
If I run `sudo oni` then it'll have permission to access the `out.log` file and launches fine.

Or, I can fix this by running
```
$ sudo touch /usr/lib/node_modules/oni-vim/cli/out.log
$ sudo chmod 777 /usr/lib/node_modules/oni-vim/cli/out.log
```

There are a couple things I think we could do here:
1. Don't write to out.log by default.  On a "good" execution, nothing is written to this log so it isn't needed in the majority use case.  We could maybe add a command-line flag to write to a log file when desired.  If we took this route, the command-line argument could just take the absolute path of where to write the log file and bypass this whole issue.
2. At "install-time" create this log file and set its permissions.  I'm not sure how to hook into the install process for Oni but I assume this would be the easier approach.

**Comment by extr0py** [comment]
Thanks for the issue and the investigation!

Option 1 makes sense to me. We can configure logging as a debug option or command line parameter. The current logging is a no-op anyway because most of the interesting logging happens in the render process.

**Comment by keforbes** [comment]
Fixed by #263.
